modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -54,23 +57,51 @@ object types {
   case class Weight(value: NonNegLong)
 
   case class ProgramsDistributionConfig(
-    weights: Map[Address, Weight] = Map(
-      Address("DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS") -> Weight(5L), // stardust tax primary
-      Address("DAG8VT7bxjs1XXBAzJGYJDaeyNxuThikHeUTp9XY") -> Weight(5L), // stardust tax secondary
-      Address("DAG77VVVRvdZiYxZ2hCtkHz68h85ApT5b2xzdTkn") -> Weight(20L), // soft staking
-      Address("DAG0qE5tkz6cMUD5M2dkqgfV4TQCzUUdAP5MFM9P") -> Weight(1L), // testnet
-      Address("DAG3RXBWBJq1Bf38rawASakLHKYMbRhsDckaGvGu") -> Weight(65L) // data pool
-    ),
-    remainingWeight: Weight = Weight(4L) // facilitators
+    weights: Map[Address, Weight],
+    remainingWeight: Weight
   )
 
   case class RewardsConfig(
-    programs: ProgramsDistributionConfig = ProgramsDistributionConfig(),
-    rewardsPerEpoch: SortedMap[EpochProgress, Amount] = SortedMap(
+    programs: EpochProgress => ProgramsDistributionConfig = mainnetProgramsDistributionConfig,
+    rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch
+  )
+
+  object RewardsConfig {
+    val stardustPrimary: Address = Address("DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS")
+    val stardustSecondary: Address = Address("DAG8VT7bxjs1XXBAzJGYJDaeyNxuThikHeUTp9XY")
+    val softStaking: Address = Address("DAG77VVVRvdZiYxZ2hCtkHz68h85ApT5b2xzdTkn")
+    val testnet: Address = Address("DAG0qE5tkz6cMUD5M2dkqgfV4TQCzUUdAP5MFM9P")
+    val dataPool: Address = Address("DAG3RXBWBJq1Bf38rawASakLHKYMbRhsDckaGvGu")
+
+    val mainnetProgramsDistributionConfig: EpochProgress => ProgramsDistributionConfig = {
+      case epoch if epoch < EpochProgress(1336392L) =>
```

Comment:
1336392 is more or less UTC 12:00 am on July 1 2023. Please verify.

Reply:
Now that I calculate it it's more like 10am UTC, so 2 hours earlier. I may have calculated the time for UTC+2 by mistake. I will monitor it and adjust if it will be the case that it wasn't my mistake during calculation and the consensus rushes a bit.